From fff981f6409077eefe676d3aa4f7cc4d80f7fe6d Mon Sep 17 00:00:00 2001
From: GuaiYiHu <guaiyihu@foxmail.com>
Date: Fri, 16 Oct 2026 06:34:40 +0800
Subject: [PATCH] liblog: Allow the scoped MiOne vendor interpreter to use
 linker defaults

Grant only device/xiaomi/mione_plus/compat/linker access to the static runtime logging dependency inherited from Bionic linker_defaults. No logging implementation or runtime policy changes.

Change-Id: If95cd916e3bf75f59e6303cb0c43206ca4160f56
---
 liblog/Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/liblog/Android.bp b/liblog/Android.bp
index 49a26f8c..3157fa0e 100644
--- a/liblog/Android.bp
+++ b/liblog/Android.bp
@@ -180,6 +180,7 @@ cc_library_static {
     ],
     visibility: [
         "//bionic/linker",
+        "//device/xiaomi/mione_plus/compat/linker",
         "//frameworks/libs/native_bridge_support/android_api/linker",
         "//system/apex/libs/libapexutil",
         "//system/core/debuggerd",
